src/SignhostClient.php

```diff
@@ -47,7 +47,6 @@ public function __construct(
         array $requestOptions = []
     ) {
         $this->headers = [
-            'Content-Type: application/json',
```

Comment:
headers[0] is replaced in later code ('$headers[0] = 'Content-Type: application/pdf';'), so this is a bug.

That code will now replace the 'Application' header.

Solution: Replace numeric keys by the header name in all $headers array and no longer hardcode their magic order.